sdk/communication/AzureCommunicationChat/README.md

```diff
@@ -385,20 +358,19 @@ client.listChatMessages(chatThreadId: threadId, withOptions: options) { result,
 
 #### Update a message
 
-Use the `update` method to update a message in a thread.
+Use the `update` method of `ChatThreadClient` to update a message in a thread.
 
 - `UpdateChatMessageRequest` is the model to pass to this method.
 - `priority` is the chat message priority `ChatMessagePriority`, such as 'Normal' or 'High', if not specified, 'Normal' will be set.
 - `content` is the message content to be updated.
-- `chatThreadId` is the unique ID of the thread.
 - `chatMessageId` is the unique ID of the message.
 
 ```swift
 let message = UpdateChatMessageRequest(
     content: "A message content with update()"
 )
 
-getClient().update(chatMessage: message, chatThreadId: threadId, chatMessageId: messageId) { result, _ in
+chatThreadClient.update(chatMessage: message, chatMessageId: messageId) { result, _ in
```

Comment:
Hmm, this one starts to get a little more interesting. Maybe something like `updateMessage(withId:changes:)`? I feel like the message ID should be the first parameter to match all of the other methods. I'm not sure what I feel the UpdateChatMessageRequest parameter should be called, but `changes` isn't terrible. Opinions?

Alternatively, using the verb-only naming scheme this would be something like `update(message:changes:)` (or maybe `update(message:withChanges:)`?)

Reply:
I agree that the identifying parameter should come first. So `update(message: String, withContent: ...)` I pulled use of content from lines 369-370. I think `update(message:with:)` would also be an acceptable format for updates.